src/kv/RocksDBStore.cc

```diff
@@ -240,6 +240,35 @@ int RocksDBStore::tryInterpret(const string &key, const string &val, rocksdb::Op
   return 0;
 }
 
+int RocksDBStore::tryInterpretStatic(const string &key, const string &val, rocksdb::Options &opt)
```

Comment:
To avoid copy-paste from tryInterpret suggest to do the following instead of adding the new duplicate method:
1) make tryInterpret function static and pass optional pointer to RocksDBStore instance there.
2) Depending on the pointer presence either process the parameter (set field or do whatever needed) or silently bypass the processing.
Hence tryInterpret will be able to do both parameter validation and there setting.